source/style-guide/style/tasks.txt

```diff
@@ -96,8 +96,12 @@ Procedures
 A task contains one or more *procedures* or sets of sequential action
 steps. Consider the following guidelines when creating a procedure:
 
--  If the procedure has more than one step, use a ``steps`` file for
-   the steps. Don't use bullets, except to list choices within a step.
+-  If the procedure has more than one step, use the ``.. procedure::`` 
+   directive for the steps. Don't use bullets, except to list choices 
+   within a step.
+-  If the procedure has more than one step, follow the guidance for 
+   introducing lists to introduce the procedure. To learn more, see 
+   :ref:`lists`.
```

Comment:
Not sure if it's within the scope of this PR, but a question came up on our team recently about how to handle "nested" steps, specifically whether nested steps should be denoted as `a. b. c.`, `i, ii, iii`, or other. Do we want to include guidance on that point here?